ajax/libs/zepto/package.json

```diff
@@ -22,4 +22,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
Mistake in https://github.com/cdnjs/cdnjs/pull/10992/commits/7f80fba6fb45c6ecd06b631ca38c97e2e13ad736?